library/styleguideComponents/Playground/index.js

```diff
@@ -13,7 +13,7 @@ export default function PlaygroundRenderer(props) {
     } = props;
     const { ...rest } = previewProps;
 
-    if (name === 'Getting Started' || name === 'Overview' || name === 'Usage' || name === 'Installation') {
+    if (name === 'Getting Started' || name === 'Overview' || name === 'Usage' || name === 'Installation' || name === 'Experiences') {
```

Comment:
this was change in last PR, with the layout change there is not more need for this only check for
```name === 'GettingStarted'```
take what is in master